### PR TITLE
Add Wexflow

### PR DESCRIPTION
**Pull request by PiotrIw**
[Wexflow](https://wexflow.github.io/) is an open source and cross-platform workflow engine and automation platform that aims to automate recurring tasks. With the help of Wexflow, building automation and workflow processes become easy.

Wexflow provides a cross-platform workflow server, a backend for designing, managing and tracking workflows, sequential workflows, flowchart workflows, and approval workflows on generic business objects called records.

Wexflow automates the process of organizing, storing and manipulating files, provides more than 100+ activities, and allows custom integration with other solutions through custom activities or Wexflow API.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ A curated list of awesome open source workflow engines
 * [Titanoboa](https://titanoboa.io/) [![Stars](https://img.shields.io/github/stars/mikub/titanoboa.svg)](https://github.com/mikub/titanoboa) - Titanoboa is a platform for creating complex workflows on JVM.
 * [uTask](https://github.com/ovh/utask) [![Stars](https://img.shields.io/github/stars/ovh/utask)](https://github.com/ovh/utask/stargazers) - Automation engine that models and executes business processes declared in yaml.
 * [Workflow Engine](https://workflowengine.io) - A lightweight .NET and Java workflow engine.
+* [Wexflow - Open Source Workflow Engine](https://github.com/aelassas/wexflow) - Wexflow is an open source and cross-platform workflow engine and automation platform that aims to automate recurring tasks. With the help of Wexflow, building automation and workflow processes become easy.
```
Comment (meirwah) [inline README.md:61]: note this list is by alphabetic order  , so this should be above `Workflow Engine` one

Comment (PiotrIw) [reply README.md:60]: corrected :)

**Comment by meirwah** [comment]
@PiotrIw can you fix conflicts ?

**Comment by PiotrIw** [comment]
@meirwah - conflicts fixed :)

**Review by meirwah — APPROVED** [review]

**Pull request merged**